uiowa/uiowa
# Hero Video Cookie

Relates to #1680 

## To Test

- Go to https://uiowa.local.drupal.uiowa.edu/ and try pausing the video. Reload or revisit the page after navigating somewhere else. Video should stay paused and button should be showing the play icon. Click the play button to resume video. Go somewhere and come back. Video should be playing.

- Check the cookie via Chrome's Dev Tools > Application > Cookies (refresh)

- Make sure Safari doesn't pull any funny business...
